src/main/scala/com/actian/spark_vector/sql/SparkSqlTable.scala

```diff
@@ -0,0 +1,30 @@
+package com.actian.spark_vector.sql
+
+import java.util.concurrent.atomic.AtomicLong
+import org.apache.spark.sql.DataFrame
+
+sealed trait SparkSqlTable {
+  def tableName: String
+  def quotedName = sparkQuote(tableName)
```

Comment:
return type

(sbt scalastyle will tell you that too for public methods, which usually get into docs)